src/MainWindow.cpp

```diff
@@ -3434,3 +3435,11 @@ void MainWindow::moveDocksTo(Qt::DockWidgetArea area)
     tabifyDockWidget(ui->dockLog, ui->dockSchema);
     tabifyDockWidget(ui->dockLog, ui->dockRemote);
 }
+
+void MainWindow::clearRecentFiles()
+{
+  Settings::clearRecentFiles();
+
+  for (int i =0; i < MaxRecentFiles; ++i) 
+    ui->fileRecentFiles->removeAction(recentFileActs[i]);
```

Comment:
This should hide the actions and not remove them, so `updateRecentFileActions()` can still work.

Reply:
And take also into account the 4 spaces used for indentation.

Reply:
Thanks for the code review!
I will fix it as soon as possible by referring to the review!

Reply:
I think resolve this conversation by 0b3869e26af4fed7e9e815a213d0ed3e33fb763b.